src/benchmark/RemotePingPong.NET46/Program.cs

```diff
@@ -0,0 +1,282 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="Program.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2016 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Runtime;
+using System.Threading;
+using System.Threading.Tasks;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.Util.Internal;
+
+namespace RemotePingPong
+{
+    public static class Messages
+    {
+        public class Msg { public override string ToString() { return "msg"; } }
+        public class Run { public override string ToString() { return "run"; } }
+        public class Started { public override string ToString() { return "started"; } }
+    }
+
+    internal class Program
+    {
+        public static uint CpuSpeed()
+        {
+#if THREADS
+            var mo = new System.Management.ManagementObject("Win32_Processor.DeviceID='CPU0'");
+            var sp = (uint)(mo["CurrentClockSpeed"]);
+            mo.Dispose();
+            return sp;
+#else
+            return 0;
+#endif
+        }
+
+        public static Config CreateActorSystemConfig(string actorSystemName, string ipOrHostname, int port)
+        {
+            var baseConfig = ConfigurationFactory.ParseString(@"
+                akka {
+              actor.provider = remote
+              loglevel = ERROR
+              suppress-json-serializer-warning = on
+              log-dead-letters = off
+
+              remote {
+                log-remote-lifecycle-events = off
+
+                dot-netty.tcp {
+                    port = 0
+                    hostname = ""localhost""
+                }
+              }
+            ");
+
+            var bindingConfig =
+                ConfigurationFactory.ParseString(@"akka.remote.dot-netty.tcp.hostname = """ + ipOrHostname + @"""")
+                    .WithFallback(ConfigurationFactory.ParseString(@"akka.remote.dot-netty.tcp.port = " + port));
+
+            return bindingConfig.WithFallback(baseConfig);
+        }
+
+        private static void Main(params string[] args)
+        {
+            Process.GetCurrentProcess().PriorityClass = ProcessPriorityClass.High;
```

Comment:
Same as we do with NBench - give this process priority with the OS scheduler while it runs. Doesn't guarantee much on a busy system but I find it helps with noise reduction.